openapi/components/schemas/Websites/WebsiteId.yaml

```
@@ -0,0 +1,5 @@
+description: |-
+  Unique website identifier string. A website is where an
+  organization obtains a customer through a subscription. For more
+  information, see
+  [Obtain an organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
```

```suggestion
type: string
description: |-
  Unique website identifier string. A website is where an
  organization obtains a customer through a subscription. For more
  information, see
  [Obtain an organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
maxLength: 50
example: 4f6cf35x-2c4y-483z-a0a9-158621f77a21
```

@noon-dawg [This](https://github.com/Rebilly/api-definitions/pull/1073/commits/12030886e46bfa0567ec8d6ec2ee0738e4e3fded) commit doesn't seem correct.

Please check customerId schema: https://github.com/Rebilly/api-definitions/blob/main/openapi/components/schemas/Cust

---

Thanks for your help @ashkarpetin. I've removed references to ResourceId in [this commit ](https://github.com/Rebilly/api-definitions/pull/1073/commits/58d2fe96415b57fdeda5ca4aeb883e6b69c60f0d)